website/content/docs/configuration/server.mdx

```diff
@@ -200,15 +200,11 @@ server {
   rejoin the cluster.
 
 - `root_key_gc_interval` `(string: "10m")` - Specifies the interval between
```

Comment:
Note: I'm intentionally leaving the name here because we'll want to use this interval for GC once we circle back to it, and I don't want to have to add a new interval for that too. Same applies to the name of the core job emitted by the leader.